org.eclipse.xtext.xbase.lib.tests/xtend-gen/org/eclipse/xtext/xbase/tests/lib/ArrayExtensionsCharTest.java

```diff
@@ -3,7 +3,7 @@
  * This program and the accompanying materials are made available under the
  * terms of the Eclipse Public License 2.0 which is available at
  * http://www.eclipse.org/legal/epl-2.0.
- *
+ * 
```

Comment:
hmm that space is not there in the original template. does this come from xtend or from the comment in the xtend file itself

Reply:
https://www.eclipse.org/projects/handbook/#ip-copyright-headers